.travis.yml

```diff
@@ -1,14 +1,23 @@
 language: node_js
 node_js:
-  - 7
-  - 6
+- 7
+- 6
 script: yarn run travis
 cache: yarn
 env:
-  - CXX=g++-4.8
+- CXX=g++-4.8
 addons:
   apt:
     sources:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
+deploy:
```

Comment:
this section is generated by travis cli, see https://docs.travis-ci.com/user/deployment/releases/ for docs